### PR TITLE
Docs Update

### PR DESCRIPTION
**Pull request by deke997**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs Update

* **Other information**:
Adds doc for installing via Bash on Windows, which allows windows users to use TA-lib.
Adds a section to the docs covering common issues with Windows Installations

### CODE REVIEW

**Comment by askmike** [comment]
Awesome stuff, would you mind adding that:

- This is a *very advanced guide* just for people who need TA-lib on windows.
- This should be temporary until we can add #708.

Thanks!

**Comment by deke997** [comment]
Done

File: docs/installation/installing_gekko_on_windows.md [hunk 1]
```diff
@@ -46,4 +50,32 @@ Your browser should automatically open with the UI. If it doesn't, manually brow
...
+gyp ERR! build error
+gyp ERR! stack Error: C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe failed with exit code: 1
+gyp ERR! stack at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:269:23)
+gyp ERR! stack at ChildProcess.emit (events.js:110:17)
+gyp ERR! stack at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
+gyp ERR! System Windows_NT 6.3.9600
+gyp ERR! command "node" "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" "rebuild"
+`
+
+#### Suggested action:
```
Comment (askmike) [inline docs/installation/installing_gekko_on_windows.md]: This error should not pop up if you are using the LTS version of Nodejs and running `npm install`. Are you sure it is for you?

**Comment by askmike** [comment]
@deke997 see comment!

**Comment by Ruthless21** [comment]
root@DESKTOP-69Q6PB4:# npm install talib@1.0.2
: not foundram Files/nodejs/npm: 3: /mnt/c/Program Files/nodejs/npm:
: not foundram Files/nodejs/npm: 5: /mnt/c/Program Files/nodejs/npm:
/mnt/c/Program Files/nodejs/npm: 6: /mnt/c/Program Files/nodejs/npm: Syntax error: word unexpected (expecting "in")
root@DESKTOP-69Q6PB4:#

File: docs/installation/installing_gekko_on_windows.md (hunk 1 above)
Comment (deke997) [reply docs/installation/installing_gekko_on_windows.md]: I was using the LTS version of Nodejs when I encountered this. Although, I used `npm install --only=production`. I can try to reproduce it if you want (and maybe open up a separate issue to discuss it..?)

**Comment by deke997** [comment]
@Ruthless21 

This looks like a common issue that arises if node is installed on both Windows and WSL (see [here](https://github.com/Microsoft/BashOnWindows/issues/1896) and [here](https://www.reddit.com/r/node/duplicates/650g4i/running_node_on_wsl/)). Luckily, the fix is easy: simply close and re-open bash. The path references don't always update until you restart bash.

Let me know if this works for you, and I'll add it to the docs.

File: docs/installation/installing_gekko_on_windows.md (hunk 1 above)
Comment (askmike) [reply docs/installation/installing_gekko_on_windows.md]: Yes please! Do you mind keeping this out for now?

Comment (deke997) [reply docs/installation/installing_gekko_on_windows.md]: Sure

**Comment by askmike** [comment]
Great stuff! Thanks :)

**Pull request merged**